python/graphscope/client/session.py

```diff
@@ -443,11 +439,13 @@ def close(self):
                 self._pod_name_list = []
 
     def close_interactive_instance(self, instance):
+        """Close a interactive instance."""
```

Comment:
I think `close_interactive_instance`, `close_learning_instance` and `deregister_default` shouldn't be public method of `Session`.